res/configuration.toml

```diff
@@ -1,68 +1,101 @@
+Type="edgex-messagebus"
+  [Trigger.EdgexMessageBus]
+  Type = "redis"
+    [Trigger.EdgexMessageBus.SubscribeHost]
+    Host = "localhost"
+    Port = 6379
+    Protocol = "redis"
+    SubscribeTopics="edgex/events/#/#/#/ROAccessReport,edgex/events/#/#/#/ReaderEventNotification"
+    [Trigger.EdgexMessageBus.Optional]
+    authmode = "usernamepassword"  # required for redis messagebus (secure or insecure).
+    secretname = "redisdb"
+    # Default MQTT Specific options that need to be here to enable evnironment variable overrides of them
+    # Client Identifiers
+    ClientId ="app-rules-engine"
+    # Connection information
+    Qos          =  "0" # Quality of Sevice values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
+    KeepAlive    =  "10" # Seconds (must be 2 or greater)
+    Retained     = "false"
+    AutoReconnect  = "true"
+    ConnectTimeout = "5" # Seconds
+    # TLS configuration - Only used if Cert/Key file or Cert/Key PEMblock are specified
+    SkipCertVerify = "false"
 
-# Every device(reader) + antenna port represents a tag location and can be assigned an alias
-# such as Freezer, Backroom etc. to give more meaning to the data. The default alias set by
-# the application has a format of <deviceName>_<antennaId> e.g. Reader-10-EF-25_1 where
-# Reader-10-EF-25 is the deviceName and 1 is the antennaId.
-# See also: https://github.com/edgexfoundry/app-rfid-llrp-inventory#setting-the-aliases
-#
-# In order to override an alias, set the default alias as the key, and the new alias as the value you want, such as:
-# Reader-10-EF-25_1 = "Freezer"
-# Reader-10-EF-25_2 = "Backroom"
-[Aliases]
+[AppCustom]
+  # Every device(reader) + antenna port represents a tag location and can be assigned an alias
+  # such as Freezer, Backroom etc. to give more meaning to the data. The default alias set by
+  # the application has a format of <deviceName>_<antennaId> e.g. Reader-10-EF-25_1 where
+  # Reader-10-EF-25 is the deviceName and 1 is the antennaId.
+  # See also: https://github.com/edgexfoundry/app-rfid-llrp-inventory#setting-the-aliases
+  #
+  # In order to override an alias, set the default alias as the key, and the new alias as the value you want, such as:
+  # Reader-10-EF-25_1 = "Freezer"
+  # Reader-10-EF-25_2 = "Backroom"
+  [AppCustom.Aliases]
 
-# See: https://github.com/edgexfoundry/app-rfid-llrp-inventory#configuration
-[ApplicationSettings]
-DeviceServiceName = "edgex-device-rfid-llrp"
-DeviceServiceURL = "http://localhost:49989/"
-MetadataServiceURL = "http://localhost:48081/"
-AdjustLastReadOnByOrigin = "true"
-DepartedThresholdSeconds = "600"
-DepartedCheckIntervalSeconds = "30"
-AgeOutHours = "336"
-MobilityProfileThreshold = "6"
-MobilityProfileHoldoffMillis = "500"
-MobilityProfileSlope = "-0.008"
+  # See: https://github.com/edgexfoundry/app-rfid-llrp-inventory#configuration
+  [AppCustom.AppSettings]
+  DeviceServiceName = "edgex-device-rfid-llrp"
+  DeviceServiceURL = "http://localhost:49989/"
```

Comment:
```suggestion
  DeviceServiceName = "device-rfid-llrp"
  DeviceServiceURL = "http://localhost:59989/"
```

Reply:
Goes away once we switch to using the Command Client

Reply:
swithced to command client and removed them